dolphinscheduler-standalone-server/src/main/resources/application.yaml

```diff
@@ -149,6 +149,8 @@ master:
   # kill yarn/k8s application when failover taskInstance, default true
   kill-application-when-task-failover: true
   worker-group-refresh-interval: 10s
+  # This attribute is mainly supported by task metric, allowing task instance name to be used as a tag.
+  support-task-name-tag-metric: true
```

Comment:
Since the number of task keeps increasing, we need clean-up mechanism for metrics tagged with task id. Otherwise, users may face memory issue. You may refer to what we did in this PR https://github.com/apache/dolphinscheduler/pull/13640